scipy/stats/_morestats.py

```diff
@@ -12,12 +12,12 @@
 from scipy import optimize
 from scipy import special
 from scipy._lib._bunch import _make_tuple_bunch
-from scipy._lib._util import _rename_parameter
+from scipy._lib._util import (_rename_parameter, _contains_nan)
 
 from . import _statlib
 from . import _stats_py
 from ._fit import FitResult
-from ._stats_py import (find_repeats, _contains_nan, _normtest_finish,
+from ._stats_py import (find_repeats, _normtest_finish,
                         SignificanceResult)
```

Comment:
While we're at it...
```suggestion
from ._stats_py import find_repeats, _normtest_finish, SignificanceResult
```